navigation/libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/BaseTest.java

```diff
@@ -10,8 +10,9 @@
 
 public class BaseTest {
   public static final double DELTA = 1E-10;
-  private static final String BASE_PATH = "/res/";
+  public static final double LARGE_DELTA = 0.1;
```

Comment:
@zugaldia added a larger delta to offset the difference between API and calculated distances.